src/components/MainParticipantInfo/MainParticipantInfo.tsx

```diff
@@ -24,11 +25,11 @@ const useStyles = makeStyles((theme: Theme) => ({
     background: 'rgba(0, 0, 0, 0.5)',
     color: 'white',
     padding: '0.1em 0.3em 0.1em 0',
-    fontSize: '1.2em',
     display: 'inline-flex',
     '& svg': {
       marginLeft: '0.3em',
     },
+    marginRight: '0.4em',
```

Comment:
I think the network container increased the height of the `identity` div just a little bit, which is totally fine. But it looks like things aren't centered vertically now. Let's add `align-items: 'center',` to the `identity` class.

Before:
![image](https://user-images.githubusercontent.com/12685223/115766961-d1b11580-a365-11eb-8664-7198c019c08c.png)

After: 
![image](https://user-images.githubusercontent.com/12685223/115767007-df669b00-a365-11eb-9e51-682a423f0cbb.png)